pkgs/tools/misc/turbo/default.nix

```diff
@@ -64,7 +64,7 @@ rustPlatform.buildRustPackage rec {
   ];
   RELEASE_TURBO_CLI = "true";
 
-  cargoSha256 = "ENw6NU3Fedd+OJEEWgL8A54aowNqjn3iv7rxlr+/4ZE=";
+  cargoSha256 = "sha256-swq8cOAM0AOvFtyHLyJgi/ngZ8vxGxylINMFzJH/Pd8=";
   RUSTC_BOOTSTRAP = 1;
   nativeBuildInputs = [
     pkg-config
```

Comment:
Please add CoreServices for darwin to fix the darwin build

Reply:
No idea about any of that tbh... I grepped nixpkgs and copied something in https://github.com/NixOS/nixpkgs/pull/217476/commits/23f14c7f98f7d74b370f7870a8024eb2f585047d , is that correct?

Reply:
You need to add
```
inherit (darwin.apple_sdk.frameworks) CoreServices;
```
to `pkgs/top-level/all-packages.nix`.

Reply:
done in 854da6f0bc17c78c1b18f1f5763c3acfe787e586